samples/snippets/src/main/java/com/example/speech/SpeechModelAdaptationBeta.java

```diff
@@ -0,0 +1,145 @@
+      LocationName parent = LocationName.of(projectId, location);
+
+      // Create the custom class
+      CreateCustomClassRequest classRequest =
+          CreateCustomClassRequest.newBuilder()
+              .setParent(parent.toString())
+              .setCustomClassId(customClassId)
+              .setCustomClass(
+                  CustomClass.newBuilder()
+                      .addItems(ClassItem.newBuilder().setValue("sushido"))
+                      .addItems(ClassItem.newBuilder().setValue("altura"))
+                      .addItems(ClassItem.newBuilder().setValue("taneda"))
+                      .build())
+              .build();
+      CustomClass classResponse = adaptationClient.createCustomClass(classRequest);
+
+      // Create the phrase set
+      CreatePhraseSetRequest phraseRequest =
+          CreatePhraseSetRequest.newBuilder()
+              .setParent(parent.toString())
+              .setPhraseSetId(phraseSetId)
+              .setPhraseSet(
+                  PhraseSet.newBuilder()
+                      .setBoost(10)
+                      .addPhrases(
+                          Phrase.newBuilder()
+                              .setValue(
+                                  String.format("Visit restaurants like %s%n", customClassId)))
+                      .build())
+              .build();
+      PhraseSet phraseResponse = adaptationClient.createPhraseSet(phraseRequest);
+
+      // Next section shows how to use the newly created custom class and phrase set
+      // to send a transcription request with speech adaptation
+
+      // Speech adaptation configuration
+      SpeechAdaptation speechAdaptation =
+          SpeechAdaptation.newBuilder()
+              .addCustomClasses(classResponse)
+              .addPhraseSets(phraseResponse)
+              .build();
+
+      // Initialize client that will be used to send requests. This client only needs to be created
+      // once, and can be reused for multiple requests. After completing all of your requests, call
+      // the "close" method on the client to safely clean up any remaining background resources.
+      try (SpeechClient speechClient = SpeechClient.create()) {
+
+        // The path to the audio file to transcribe
+        // gcsUri URI for audio file in Cloud Storage, e.g. gs://[BUCKET]/[FILE]
+
+        // Builds the sync recognize request
+        RecognitionConfig config =
+            RecognitionConfig.newBuilder()
+                .setEncoding(AudioEncoding.FLAC)
+                .setSampleRateHertz(16000)
+                .setLanguageCode("en-US")
+                .setAdaptation(speechAdaptation) // Set the adaptation object
+                .build();
+        RecognitionAudio audio = RecognitionAudio.newBuilder().setUri(gcsUri).build();
+
+        // Performs speech recognition on the audio file
+        RecognizeResponse response = speechClient.recognize(config, audio);
```

Comment:
What if there is an exception in calling this? We are not showing how to catch the exception and what kind of exception to catch here.

Reply:
@stephaniewang526 would you please provide a recommendation for doing exception handling for this sample?

Reply:
Here is an [example](https://github.com/googleapis/java-bigquery/blob/ba0cfc2cf76d9fa16c2b75870338414d38e1722b/samples/snippets/src/main/java/com/example/bigquery/CreateDataset.java#L45)